relayer/chainproxy/chainproxy.go

```diff
@@ -53,13 +54,13 @@ type ChainProxy interface {
 	GetCache() *performance.Cache
 }
 
-func GetChainProxy(nodeUrl string, nConns uint, sentry *sentry.Sentry, pLogs *PortalLogs) (ChainProxy, error) {
+func GetChainProxy(nodeUrl string, nConns uint, sentry *sentry.Sentry, pLogs *PortalLogs, flagSet *pflag.FlagSet) (ChainProxy, error) {
```

Comment:
this is soon to be deprecated needs to be done also in protocol/chainlib/chainproxy